_pyntcore/src/py2value.cpp

```diff
@@ -81,11 +81,7 @@ std::shared_ptr<nt::NetworkTableValue> py2ntvalue(py::handle h) {
   // check the first item
   auto i1 = seq[0];
   if (py::isinstance<py::bool_>(i1)) {
-    std::vector<int> v;
-    v.reserve(seq.size());
-    for (size_t i = 0; i < seq.size(); i++) {
-      v.push_back(seq[i].cast<int>());
-    }
+    auto v = h.cast<std::vector<int>>();
```

Comment:
why not follow the form of the rest and return the cast directly?

Reply:
I don't understand the question. This _is_ written like the rest.

Reply:
You create a variable, the others don't have a variable

Reply:
oh I guess this lower section is different from above. carry on